src/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -2105,6 +2105,9 @@ public sealed partial class Encoder
         public static readonly System.Drawing.Imaging.Encoder ScanMethod;
         public static readonly System.Drawing.Imaging.Encoder Transformation;
         public static readonly System.Drawing.Imaging.Encoder Version;
+        public static readonly System.Drawing.Imaging.Encoder ColorSpace;
```

Comment:
Are these needed in order to fix the bugs? Can we wait for the APIs to be approved first?

Reply:
They are not needed (I used them in a unit test, but since they don't exist on NetFx, I had to revert that change).